Makefile

```diff
@@ -199,7 +199,17 @@ check-clean-work-tree:
 .PHONY: cli-install
 cli-install:
 	@echo "Installing odigos from source. version: $(ODIGOS_CLI_VERSION)"
-	cd ./cli ; go run -tags=embed_manifests . install --version $(ODIGOS_CLI_VERSION)
+	cd ./cli ; go run -tags=embed_manifests . install --version $(ODIGOS_CLI_VERSION) --nowait
+
+.PHONY: cli-uninstall
+cli-uninstall:
+	@echo "Uninstalling odigos from source. version: $(ODIGOS_CLI_VERSION)"
+	cd ./cli ; go run -tags=embed_manifests . uninstall
+
+.PHONY: cli-uninstall
+cli-uninstall:
+	@echo "Uninstalling odigos from source. version: $(ODIGOS_CLI_VERSION)"
+	cd ./cli ; go run -tags=embed_manifests . uninstall
```

Comment:
```suggestion
```

the `cli-uninstall` target is already declared below these, this makes 3 of them

Reply:
Thanks, will sort it now 👍